ts/screens/wallet/payment/components/TransactionSummary.tsx

```diff
@@ -96,63 +61,34 @@ const InfoButton = (props: { onPress: () => void }) => (
 );
 
 type RowProps = Readonly<{
-  axis: "horizontal" | "vertical";
   title: string;
-  subtitle?: string;
-  icon?: React.ReactNode;
-  placeholder?: React.ReactNode;
+  value?: string;
+  icon?: IOIcons;
   isLoading?: boolean;
-  hideSeparator?: boolean;
+  placeholder?: React.ReactElement;
+  action?: React.ReactNode;
   onPress?: () => void;
 }>;
 
 export const TransactionSummaryRow = (
   props: React.PropsWithChildren<RowProps>
 ): React.ReactElement | null => {
-  if (!props.isLoading && !props.subtitle) {
-    return null;
-  }
```

Comment:
If we remove this guard, the component will be displayed even if the `value` is empty.
| old | new |
| --- | --- |
| <img src="https://github.com/pagopa/io-app/assets/29163287/bf6e0414-dc48-4ebf-a502-59692e0ca64e" width="250" /> | <img src="https://github.com/pagopa/io-app/assets/29163287/04698a84-9df1-4b30-8aa1-957c434f4285" width="250" /> |

Reply:
Good catch! Addressed into de0af2b4f218ad8a3c35cf4d510f75f1ada86106